### ISSUE TITLE
When edit mode enabled for a row, empty cell jumps up vertically and is stuck there

### ISSUE THREAD

**Issue by davidrleonard** [vaadin/px-data-grid#394]
### Steps to reproduce

1. Create a px-data-grid tag
2. Add a custom `columns` property and define some editable columns:

```
[
  {
    name: 'Name',
    path: 'name',
    type: 'string',
    editable: false
  },
  {
    name: 'Age',
    path: 'age',
    type: 'number',
    editable: true,
    renderer: 'px-data-grid-number-renderer'
  },
  {
    name: 'City',
    path: 'city',
    type: 'string',
    editable: true,
    renderer: 'px-data-grid-string-renderer'
  }
]
```

3. Add some `tableData`. Leave the age empty. We want the user to use the edit button to fill the age in:

```
[
  {
    name: 'John',
    city: 'San Francisco'
  }
]
```

4. Click on the edit button to the right of the row.
5. The age input will jump up vertically. Fill in an age anyway, try 50.
6. Click the save button to the right of the row to save the changes.

**Expected behavior:** When the row becomes editable all input boxes should stay in the correct place vertically, even if they are empty. On save they should also be in the correct place.
**Actual behavior:** The empty age input jumps up vertically and becomes misaligned when the row is switched to edit mode. When the row is saved the age cell is still misaligned.

### Version

px-data-grid: v1.1.0
Browsers: Chrome 64

### Screenshots

1. Before editing, everything seems normal:

![image](https://user-images.githubusercontent.com/6456757/35073220-db983d54-fb9c-11e7-9c58-b6058e356ba8.png)

2. Edit mode enabled, age input jumps up

![image](https://user-images.githubusercontent.com/6456757/35073260-04ecddea-fb9d-11e7-8bf7-365c816fa889.png)

3. After saved, age cell is still misaligned

![image](https://user-images.githubusercontent.com/6456757/35073287-179d9eca-fb9d-11e7-9f2d-547c01eccf96.png)

**Comment by davidrleonard** [comment]
Issue is here: https://github.com/vaadin/px-data-grid/blob/62ea5dac512ce2b25c6e92dd9b98c277dc448e01/px-data-grid-renderer-mixin.html#L64

Have to dig more to figure out how to ensure there isn't a 0 value returned by measuring the input when it has no value.

**Comment by Aileen-H** [comment]
PRIORITY 1

**Comment by petrixh** [comment]
Fixed in 1.1.11
